plugins/cpp/test/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ include_directories(
   ${PROJECT_BINARY_DIR}/service/language/gen-cpp
   ${PROJECT_BINARY_DIR}/service/project/gen-cpp
   ${PROJECT_SOURCE_DIR}/model/include
+  ${PROJECT_SOURCE_DIR}/webserver/include
```

Comment:
Changes in other headers broke the test's file.
The test's header includes the service header, and as such, this subproject also needs to know how to resolve the `#include <webserver/servercontext.h>` in `cppservice.h`.